plugin/clone/src/clone_status.cc

```diff
@@ -465,6 +474,33 @@ void Status_pfs::Data::write(bool write_error) {
   status_file.close();
 }
 
+void Status_pfs::Data::write_synchronization_coordinate(
+    const Key_Value &coordinate) {
+  std::string file_name =
```

Comment:
```suggestion
  const auto file_name =
```

Reply:
would const std::string suffice? why is declaring the type directly so frowned upon? for types that are not long I personally like to read the direct type for better understanding.

Reply:
TL;DR: yes "const std::string" is OK too. 
Having said that, I believe AAA (Almost Always Auto) is the right C++ style: usually we don't care about the exact types of local variables, while we do care about the parameter and return types of methods (incidentally it is how Rust looks by default). Should we ever refactor and change the type, auto would reduce the diff churn. In rare cases where the type of a local variable matters and is not obvious from its initializer, then usually IDE support can show it.